pkg/cmd/cli/cmd/tag.go

```diff
@@ -133,7 +133,7 @@ func parseStreamName(defaultNamespace, name string) (string, string, error) {
 
 func determineSourceKind(f *clientcmd.Factory, input string) string {
 	mapper, _ := f.Object()
-	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.GroupName, Resource: input})
+	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.LegacyGroupName, Resource: input})
```

Comment:
Wasn't it working before? How come this wasn't caught in @mfojtik's api groups pr.